src/Infrastructure/Doctrine/ORM/ChannelPricingLogEntryRepository.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Infrastructure\Doctrine\ORM;
+
+use Doctrine\ORM\QueryBuilder;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+
+final class ChannelPricingLogEntryRepository extends EntityRepository implements ChannelPricingLogEntryRepositoryInterface
+{
+    public function createByChannelPricingIdListQueryBuilder(mixed $channelPricingId): QueryBuilder
+    {
+        return $this->createQueryBuilder('o')
+            ->innerJoin('o.channelPricing', 'channelPricing')
+            ->andWhere('channelPricing = :channelPricingId')
+            ->orderBy('o.id', 'DESC')
```

Comment:
```suggestion
```
IMO it should be left for the grid configuration to handle sorting

Reply:
We do not have an `id` field in the grid, so it is not possible to sort it in this manner. We could sort it using the `loggedAt` field, but it would be inaccurate if prices change at the same time.

Reply:
Pretty sure it works without a field, but IIRC there were some issues with that approach, so let's leave it as is.